gravitee-apim-rest-api/gravitee-apim-rest-api-management-v2/gravitee-apim-rest-api-management-v2-rest/src/main/java/io/gravitee/rest/api/management/v2/rest/resource/api/ApisResource.java

```diff
@@ -139,6 +141,9 @@ public ApiResource getApiResource() {
     @Inject
     private OAIToImportApiUseCase oaiToImportApiUseCase;
 
+    @Inject
+    private ApiMetadataQueryService apiMetadataQueryService;
+
```

Comment:
Do you think you could find another way to retrieve metadata ?
I'm not sure QueryServices are supposed to be used outside the service layer.
It should be either a use-case or a legacy service